server/api.ts

```diff
@@ -1104,6 +1105,33 @@ router.postAsync("/stage-stats", ensure.isAdmin, async (req, res) => {
   res.send(updatedStageStats);
 });
 
+/**
+ * POST /api/refetch-stats
+ * Refretches the stats of a batch of mp links in a stage
+ * Params:
+ *   - tourney: identifier for the tourney
+ *   - stage: identifier for the stage
+ * Returns:
+ *   - stats: the updated stage stats
+ */
+router.postAsync("/refetch-stats", ensure.isAdmin, async (req, res) => {
```

Comment:
new routes should be typed like this:
https://github.com/cory2067/gtsosu/blob/cf7c43224339e05e14eeb6cdf58e00d5a0ea089f/server/api/map.ts#L48
(and these stats routes preferably pulled out into a sub-router at some point)

Reply:
okay